build-system/pr-check/visual-diff-tests.js

```diff
@@ -36,6 +36,7 @@ function prBuildWorkflow() {
   if (buildTargetsInclude(Targets.RUNTIME, Targets.VISUAL_DIFF)) {
     timedExecOrDie('amp visual-diff --nobuild');
   } else {
+    timedExecOrDie('amp visual-diff --empty');
```

Comment:
This will need the token to be decoded.

```suggestion
    process.env['PERCY_TOKEN'] = atob(process.env.PERCY_TOKEN_ENCODED);
    timedExecOrDie('amp visual-diff --empty');
```

Needs this line at the top:

```js
const atob = require('atob');
```

Reply:
Don't forget to add the `require` for `atob`.

Reply:
You're right! This entire thread can be ignored.